### DISCUSSION TITLE
How to add maintenance banner in Harbor to announce upgrades and downtime?

### DISCUSSION THREAD

**Discussion by Mais316** [goharbor/harbor#16829] [Q&A] [answered]
How to add maintenance banner in Harbor to announce upgrades and downtime?

**Comment by wy65701436** [comment, marked as answer]
Can the read-only bannar meet it?

**Reply by Vad1mo** [reply]
There is only the "read-only" banner. IMHO it should do it.
You can change the text in the language file of harbor and add eg. "Maintenance Mode" if you are up for it.
